Added new copyright test cases

abhishak3 opened the pull request:
<!-- Delete Template sections if unneccesary -->
<!-- Add issue number here (We encourage you to create the Issue First) -->
<!-- You can also link the issue in Commit Messages -->

Issue #2299 
Added some new copyright test cases.

<!--
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!
* [x] - Checked Box
* [ ] - Unchecked Box
-->


<!--
We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged**
if your tests fail at first!
If tests do fail, click on the red `X` to learn why by reading the logs.
Thanks!
-->

<!-- Don't forget to Signoff -->

abhishak3 committed:
Added new copyright test cases

Signed-off-by: Abhishek <abhishek.pandey.3237@gmail.com>

abhishak3 committed:
added new test cases for copyright detection

Signed-off-by: abhishak3 <abhishek.pandey.3237@gmail.com>

pombredanne reviewed (commented):
Thank you ++
This is looking great. Still need to review each results in details though.

pombredanne commented on tests/cluecode/data/copyrights/long_holder_name.txt.yml:
```diff
@@ -0,0 +1,9 @@
+  - holders
+
+copyrights:
+  - Copyright (c) 2000-2021 Adolph Blaine Charles David Earl Frederick Gerald Hubert Irvin John Kenneth Lloyd Martin Nero Oliver Paul Quincy Randolph Sherman Thomas Uncas Victor William Xerxes Yancy Zeus Wolfeschlegelsteinhausenbergerdorffwelchevoralternwarengewissenhaftschaferswessenschafewarenwohlgepflegeundsorgfaltigkeitbeschutzenvonangreifendurchihrraubgierigfeindewelchevoralternzwolftausendj ...
```
I think this should be marked as an expected failure and you should not expect the very long line at the end.
Having this long line is a bug to fix IMHO

pombredanne replied:
I pushed an update in your branch

pombredanne commented on tests/cluecode/data/copyrights/long_name.txt.yml:
```diff
@@ -0,0 +1,9 @@
+  - holders
+
+copyrights:
+  - Copyright (c) 2021 Adolph Blaine Charles David Earl Frederick Gerald Hubert Irvin John Kenneth Lloyd Martin Nero Oliver Paul Quincy Randolph Sherman Thomas Uncas Victor William Xerxes Yancy Zeus Wolfeschlegelsteinhausenbergerdorffwelchevoralternwarengewissenhaftschaferswessenschafewarenwohlgepflegeundsorgfaltigkeitbeschutzenvonangreifendurchihrraubgierigfeindewelchevoralternzwolftausendjahres ...
```
Same as before: I think this should be marked as an expected failure and you should not expect the very long line at the end. Having this long line is a bug to fix IMHO

pombredanne replied:
I pushed an update in your branch

pombredanne reviewed (changes requested):
Thanks! See some minor changes to mark some tests as failing for now. 
See https://github.com/nexB/scancode-toolkit/blob/ebb55c341bc6ba644d2be0e531ffbfc2948bca47/tests/cluecode/data/generated/copyright_30.txt.yml#L9 for an example.

pombredanne committed:
Do not detect copyrights in very long name

Nouns NNs and Prper nouns NNPs should be limited in length.
For now these texts are expected to fail until we add a limit, say of
50 characters.

Signed-off-by: Philippe Ombredanne <pombredanne@nexb.com>

pombredanne reviewed (approved):
LGTM. Thanks! metrging now
